### ISSUE TITLE
Auto Adjusting Column Length and Row Height 

### ISSUE THREAD

**Issue by jmcgrath207** [DeepSpace2/StyleFrame#34]
Hi! Thanks for contributing to StyleFrame. Please make sure to mention the Python, StyleFrame, pandas and openpyxl versions you are using.

I am having an issue with the excel sheet auto adjusting the column to the widest length, below is some sample code I am having an issue with.

```
import pandas as pd
from StyleFrame import StyleFrame, Styler, utils

df = pd.DataFrame({'WW': ['WWWW', 'WWWWWWWWWWWWWWWWWW']})

sf = StyleFrame(df)
ew = StyleFrame.ExcelWriter('pandas_simple.xlsx')
sf.to_excel(ew)
ew.save()
```

Here are the versions I am using

Python: 3.5.2
StyleFrame: 1.3.1
Pandas 0.20.3
openpyxl: 2.2.5

**Comment by DeepSpace2** [comment]
What is the current output you're getting vs what you're actually getting?

You have several options: 

- Use `sf.set_column_width`. 
- Pass `shrink_fit` and `wrap_text` with `False` for `Styler`:
   `sf = StyleFrame(df, Styler(shrink_to_fit=False, wrap_text=False))`

**Comment by jmcgrath207** [comment]
Thanks for getting back to me so quickly


Running this gives me

```
import pandas as pd
from StyleFrame import StyleFrame, Styler, utils

df = pd.DataFrame({'WW': ['WWWW', 'WWWWWWWWWWWWWWWWWW']})

sf =  StyleFrame(df, Styler(shrink_to_fit=False, wrap_text=False))
ew = StyleFrame.ExcelWriter('pandas_simple.xlsx')
sf.to_excel(ew)
ew.save()
```


![image](https://user-images.githubusercontent.com/8764013/34629560-2defd184-f22e-11e7-8421-4efbf332a484.png)


I am wanting something like 
![image](https://user-images.githubusercontent.com/8764013/34629392-8a1fb3a8-f22d-11e7-8cca-1e8d710a9679.png)


I am displaying the excel sheet with libre calc on Linux mint 18.2 if that helps

**Comment by DeepSpace2** [comment]
I'll need to look into this.

As a workaround, a possible solution would be using `sf.set_column_width_dict` to set each column's width according to the longest string in it multiplied by some factor:

```
df = pd.DataFrame({'WW': ['WWWW', 'WWWWWWWWWWWWWWWWWW'],
                   'ZZ': ['ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ', 'ZZ']})
sf = StyleFrame(df)
factor = 2.5
sf.set_column_width_dict({column: max(df[column].str.len() * factor) for column in df.columns})
ew = StyleFrame.ExcelWriter('pandas_simple.xlsx')
sf.to_excel(ew).save()
```

**Comment by DeepSpace2** [comment]
Solved in version 1.4. See information about `best_fit` argument of `to_excel` method and the note at the bottom: http://styleframe.readthedocs.io/en/latest/api_documentation.html#to-excel

**Comment by jnw1** [comment]
> Solved in version 1.4. See information about `best_fit` argument of `to_excel` method and the note at the bottom: http://styleframe.readthedocs.io/en/latest/api_documentation.html#to-excel

I found it's not working when a cell include multiple lines, always one line's height even if I use both "wrap_text=True" and  best_fit=columns. And width is too large since it compress all words in a line.
![image](https://user-images.githubusercontent.com/108532139/196031853-8b94af22-6fbc-4086-a0e0-754e0d6b5b3d.png)

**Comment by DeepSpace2** [comment]
@jnw1 I'm not sure I follow, can you please explain again? Also, see #87.
